package.json

```diff
@@ -14,16 +14,16 @@
     "test:lint": "standard",
     "test:unit": "tape -r './src/test/unit/setup' 'src/@(web|config)/**/*.test.js'",
     "test:unit:coverage": "nyc npm run test:unit",
-    "test:acceptance": "mkdir -p build/reports && cross-env TESTS=acceptance cucumber-js src/test/acceptance/features/ --tags 'not @ignore' --require 'src/test/common/steps/*.js' --require 'src/test/acceptance/config' -f json:build/reports/acceptance-report.json",
-    "test:acceptance-no-wiremock": "mkdir -p build/reports && cross-env TESTS=acceptance cucumber-js src/test/acceptance/features/ --tags 'not @ignore' --tags 'not @RequiresWiremock' --require 'src/test/common/steps/*.js' --require 'src/test/acceptance/config' -f json:build/reports/acceptance-report.json",
-    "test:acceptance:debug": "mkdir -p build/reports && cross-env TESTS=acceptance node $NODE_DEBUG_OPTION node_modules/cucumber/bin/cucumber-js src/test/acceptance/features/ --tags 'not @ignore' --require 'src/test/common/steps/*.js' --require 'src/test/acceptance/config' -f json:build/reports/acceptance-report.json",
+    "test:acceptance": "mkdir -p build/reports && cross-env TESTS=acceptance cucumber-js src/test/acceptance/features/navigation.feature --tags 'not @ignore' --require 'src/test/common/steps/*.js' --require 'src/test/acceptance/config' --require 'src/test/common/timeout.js' -f json:build/reports/acceptance-report.json",
```

Comment:
`navigation.feature` has to come out again - that was there for local testing. 
Rather than making this same change in several lines here, it might be better if there was a script in the config directory that imported the script from common.

Reply:
Ok. I'll add that to the next PR so this one can get through without an extra file and a rebuild.

Reply:
I'd get other opinions on that before actually making changes. There's an argument that making it explicit is good - it took me a while to figure how how it was being executed for the acceptance tests.